packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,660 @@
+---
+title: Input - Form
+package: '@twilio-paste/form'
+description: An input allows users to enter text on a single line.
+---
+
+import {graphql} from 'gatsby';
+import {FormLabel, FormInput, FormHelperText} from '@twilio-paste/form';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/form/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/form"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/form/input/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Input"
+  categoryRoute={SidebarCategoryRoutes.FORM}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/form"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-form--input-options"
+  abstractUrl="https://share.goabstract.com/85c81258-90b9-49c8-b34b-9418da5f2f00"
```

Comment:
Abstract link needs updating. This is the typography collections. I think the correct link is: https://share.goabstract.com/6cc5fc40-3408-4cfd-b1d0-103aea01601d

Reply:
Good catch! Changed to: https://share.goabstract.com/6cc5fc40-3408-4cfd-b1d0-103aea01601d